src/App.js

```diff
@@ -6,10 +6,10 @@ import Footer from "./Components/Footer";
 import Header from "./Components/Header";
 
 function App() {
-  const [isLoading, setIsLoading] = useState(true);
+  // const [isLoading, setIsLoading] = useState(true);
```

Comment:
can delete this; we want data-fetching components to have their own loading states